metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -75,6 +76,59 @@ def get_sql_alchemy_url(self):
         return url
 
 
+class OracleInspectorObjectWrapper:
+    """
+    Inspector class wrapper to resolve linear issue
+    https://linear.app/acryl-data/issue/ACR-3367/github-oracle-ingestion-not-ingesting-system-table-space-5167
+    """
+
+    def __init__(self, inspector_instance: Inspector):
+        self._inspector_instance = inspector_instance
+        self.log = logging.getLogger(__name__)
```

Comment:
in python, normally you just have a variable at the top of the file called `logger = logging.getLogger(__name__)` and just use that - no need for an instance variable